packages/admin/admin/src/mui/menu/Menu.tsx

```diff
@@ -1,37 +1,120 @@
-import { ComponentsOverrides, Drawer, DrawerProps, PaperProps, Theme } from "@mui/material";
-import { WithStyles, withStyles } from "@mui/styles";
+import Drawer from "@mui/material/Drawer";
+import { PaperProps } from "@mui/material/Paper";
+import { ComponentsOverrides, css, styled, Theme, useThemeProps } from "@mui/material/styles";
+import { ThemedComponentBaseProps } from "helpers/ThemedComponentBaseProps";
 import * as React from "react";
 import { useHistory } from "react-router";
 
 import { MasterLayoutContext } from "../MasterLayoutContext";
 import { MenuContext } from "./Context";
-import { MenuClassKey, styles } from "./Menu.styles";
 
-export interface MenuProps {
+export type MenuClassKey = "permanentDrawer" | "temporaryDrawer" | "permanent" | "temporary" | "open" | "closed";
```

Comment:
The new class-keys are a bit confusing...

Should we maybe keep `drawer` so it can continue to be used, as is it currently still used on both `PermanentDrawer` and `TemporaryDrawer`? 

Also `permanentDrawer`, `temporaryDrawer`, `permanent` and `temporary` are all not being used correctly...
Maybe we should remove `permanent` and `temporary`, mention in the changeset that they were renamed to `permanentDrawer` and `temporaryDrawer` and actually use them in thier respective component's `overridesResolver` in addition to `drawer`.